UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.ts

```diff
@@ -549,8 +550,31 @@ export class MangaReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         takeUntilDestroyed(this.destroyRef)
       ).subscribe(() => {});
 
+      this.generalSettingsForm.get('widthSlider')?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
+        //update the css width
+        const styleSheets = document.styleSheets;
```

Comment:
This isn't needed. You just need the width value from this. So:

```
this.widthOverride$ = this.generalSettingsForm.get('widthSlider')?.valueChanges.pipe(
map(val => val < -1 ? 'initial' : val + '%'),
takeUntilDestroyed(this.destroyRef)
);
```

This would give you back the string value. Then on the image you'd do style={{widthOverride$}}. 

But the thing is, the image isn't in this component. It is rendered out in the renderers. So while the control is here, it would need to populate the readerSettings and the above code would need to be in each renderer. 

You probably also need to disable the input for double page layout.